pkg/controller/provider/container/ovirt/model.go

```diff
@@ -11,22 +11,16 @@ import (
 
 //
 // Event codes.
-// NICProfileAdded   = 1122
-// NICProfileUpdated = 1124
-// NICProfileDeleted = 1126
 const (
-	DataCenterAdded   = 950
-	DataCenterUpdated = 952
-	DataCenterDeleted = 954
-	ClusterAdded      = 809
-	ClusterUpdated    = 811
-	ClusterDeleted    = 813
-	HostAdded         = 42
-	HostUpdated       = 43
-	HostDeleted       = 44
-	VmAdded           = 34
-	VmUpdated         = 35
-	VmDeleted         = 113
+	USER_ADD_CLUSTER    = 809
+	USER_UPDATE_CLUSTER = 811
+	USER_REMOVE_CLUSTER = 813
+	USER_ADD_HOST       = 42
+	USER_UPDATE_HOST    = 43
+	USER_REMOVE_HOST    = 44
+	USER_ADD_VM         = 34
+	USER_UPDATE_VM      = 35
+	USER_REMOVE_VM      = 113
```

Comment:
This change is unrelated to flattened trees.
Using all CAPS to match [documentation](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.2/html/technical_reference/appe-event_codes)